templates/express/app/entry.server.tsx

```diff
@@ -2,6 +2,7 @@ import { PassThrough } from "stream";
 import type { EntryContext } from "@remix-run/node";
 import { Response } from "@remix-run/node";
 import { RemixServer } from "@remix-run/react";
+import isbot from "isbot";
```

Comment:
This isn't a huge deal, but TS marks this import as missing in dev. I've always tried to avoid this kind of thing in the past. Not sure the best way to fix it here though since this is in our templates.

Maybe we just add `isbot` to our monorepo deps to avoid the TS warning? Just a thought. Open to your suggestions.

![Screen Shot 2022-09-29 at 3 49 27 PM](https://user-images.githubusercontent.com/92839/193155611-83bc8d18-dba9-478c-b480-4af73d790d4a.png)

Reply:
#4061 will add `isbot` to the monorepo deps, so these errors should be gone once that one's merged I think.